components/dashboard/ActionTableTooltips.js

```diff
@@ -85,35 +113,32 @@ export const phasesTooltipContent = (
     else return mergedWith.identifier;
   };
 
+  const statusDisplay = (
+    <StatusLabel color={status?.color}>{status.label}</StatusLabel>
+  );
+  // If action is merged, display merged status
   if (merged)
     return (
-      <div>
-        <h5>
-          {` ${t(
-            'actions:action-status-merged',
-            getActionTermContext(plan)
-          )}: ${getMergedName(merged, plan.id)}.`}
-        </h5>
-      </div>
-    );
-  if (!activePhase)
-    return (
-      <div>
-        {!status?.name && (
-          <h5>{t('actions:action-implementation-phase-unknown')}</h5>
-        )}
-        {status?.name}
-      </div>
+      <TooltipTitle>
+        {` ${t(
+          'actions:action-status-merged',
+          getActionTermContext(plan)
+        )}: ${getMergedName(merged, plan.id)}.`}
+      </TooltipTitle>
     );
-  if (status?.identifier === 'cancelled')
-    return (
-      <div>
-        <h5>{status.name}</h5>
-      </div>
-    );
-  return hasImplementationPhases ? (
+  // If action has no active phase or it's cancelled, or plan has no implementation phases : display only status
+  if (
+    !activePhase ||
+    status?.identifier === 'cancelled' ||
```

Comment:
Since this file isn't type checked yet, we can leave it be, but in the future we can check for legal status summary identifier by comparing to https://github.com/kausaltech/kausal-watch-ui/blob/feat%2Fshow-status-on-tooltip/common/__generated__/graphql.ts#L773-L785